test/com/facebook/buck/features/python/toolchain/impl/PythonInterpreterFromConfigTest.java

```diff
@@ -98,9 +98,9 @@ public void whenPythonPlusExtensionOnPathIsExecutableFileThenItIsUsed() throws I
   }
 
   @Test
-  public void whenPython2OnPathThenItIsUsed() throws IOException {
+  public void whenPython3OnPathThenItIsUsed() throws IOException {
```

Comment:
I think you actually want both versions of this test. The intent seems to be supporting executables that aren't `python`

Reply:
I checked the 2 to 3 migration guideline, it highlighted "PYTHON 2 IS DEAD H2 2020." and "Tell linter to check Python 3 only". So we should support Python3 only.